squirrel_hri_msgs/CMakeLists.txt

```diff
@@ -0,0 +1,44 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(squirrel_hri_msgs)
```

Comment:
please add squirrel_hri_msgs to squirrel_common metapackage: https://github.com/squirrel-project/squirrel_common/blob/indigo_dev/squirrel_common/package.xml